lib/runners/cplusplus.rb

```diff
@@ -0,0 +1,44 @@
+module Runners
+  module CPlusPlus
+    # @see https://github.com/github/linguist/blob/775b07d40c04ef6e0051a541886a8f1e30a892f4/lib/linguist/languages.yml#L532-L535
+    # @see https://github.com/github/linguist/blob/775b07d40c04ef6e0051a541886a8f1e30a892f4/lib/linguist/languages.yml#L568-L584
+    CPP_SOURCES_GLOB = "*.{c,cpp,c++,cc,cp,cxx}".freeze
+    CPP_HEADERS_GLOB = "**/*.{h,h++,hh,hpp,hxx,inc,inl,ipp,tcc,tpp}".freeze
+    private_constant :CPP_SOURCES_GLOB, :CPP_HEADERS_GLOB
+
+    def config_include_path
+      Array(config_linter[:'include-path'] || find_paths_containing_headers).map { |v| "-I#{v}" }
+    end
+
+    def cpp_file?(path)
+      path.fnmatch?(CPP_SOURCES_GLOB, File::FNM_EXTGLOB | File::FNM_CASEFOLD)
+    end
+
+    def deploy_packages
+      # select development packages and report others as warning for security concerns
+      packages = Array(config_linter[:apt])
+        .select do |pkg|
+          if pkg.match?(/-dev(=.+)?$/)
+            true
+          else
+            add_warning "Installing the package `#{pkg}` is blocked.", file: config.path_name
+            false
+          end
+        end
+
+      if packages.empty?
+        trace_writer.message "No packages to install."
```

Comment:
[nitpick] How about this? 👇 

```suggestion
        trace_writer.message "No apt packages to install."
```

Reply:
commit 035f9656607796996feb63b06b8bce200e43cd95